neural_style.py

```diff
@@ -118,25 +161,49 @@ def main():
     for iteration, image in stylize(
         network=options.network,
         initial=initial,
+        initial_noiseblend=options.initial_noiseblend,
         content=content_image,
         styles=style_images,
         iterations=options.iterations,
         content_weight=options.content_weight,
+        content_weight_blend=options.content_weight_blend,
         style_weight=options.style_weight,
+        style_layer_weight_exp=options.style_layer_weight_exp,
         style_blend_weights=style_blend_weights,
         tv_weight=options.tv_weight,
         learning_rate=options.learning_rate,
+        beta1=options.beta1,
+        beta2=options.beta2,
+        epsilon=options.epsilon,
+        pooling=options.pooling,
         print_iterations=options.print_iterations,
         checkpoint_iterations=options.checkpoint_iterations
     ):
         output_file = None
+        combined_rgb = image
         if iteration is not None:
             if options.checkpoint_output:
                 output_file = options.checkpoint_output % iteration
         else:
+            if options.preserve_colors == 1:
```

Comment:
This is some thin ice for me frankly, currently the RGB<->YUV color model transformation is performed by another tf graph, and I simply am not sure how tf will handle graph within a graph; and how will if affect the overall training (it shouldn't).
It is possible to rewrite it using core numpy I think, but then it wouldn't be adapted change :)